sqlglot/expressions.py

```diff
@@ -855,10 +855,14 @@ def isin(
         copy: bool = True,
         **opts,
     ) -> In:
+        subquery = maybe_parse(query, copy=copy, **opts) if query else None
+        if isinstance(subquery, Query) and not isinstance(subquery, Subquery):
+            subquery = subquery.subquery(copy=False)
```

Comment:
Perhaps we can always wrap if it's a `Query`? this implementation intended for `isin(query="(select 1)")` to not produce an AST where the `Select` is wrapped twice.

Reply:
```python
if subquery and not isinstance(subquery.Subquery):
    ...
```
?